chains/evm/contracts/pools/SiloedLockReleaseTokenPool.sol

```diff
@@ -33,8 +35,8 @@ contract SiloedLockReleaseTokenPool is TokenPool, ITypeAndVersion {
   /// @notice The rebalancer for unsiloed chains, which can add liquidity to the shared pool.
   address internal s_rebalancer;
 
-  /// @notice The lock box for the token pool.
-  ERC20LockBox internal immutable i_lockBox;
+  /// @notice Lock boxes keyed by chain selector. Chain selector 0 is used for unsiloed liquidity.
+  mapping(uint64 remoteChainSelector => ERC20LockBox lockBox) internal s_lockBoxes;
```

Comment:
I thought the agreement was to have the lockbox handle silos, always having 1 lockbox per lock release token pool (even if siloed)?

Reply:
On second thought, would actually prefer to keep 1 lockbox per pool the standard. Makes deployment tooling easier.